logbat/src/test/java/info/logbat/log/domain/enums/LevelTest.java

```diff
@@ -0,0 +1,61 @@
+package info.logbat.log.domain.enums;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class LevelTest {
+
+  @DisplayName("빈 문자열의 로그 레벨 문자열을 enum으로 변환하면 예외가 발생한다.")
+  @Test
+  void createLevelWithBlank() {
+    // given
+    String 빈_로그_레벨 = "";
+
+    // when & then
+    assertThatThrownBy(() -> Level.from(빈_로그_레벨))
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessage("level은 null이거나 빈 문자열일 수 없습니다.");
+  }
+
+  @DisplayName("null의 로그 레벨 문자열을 enum으로 변환하면 예외가 발생한다.")
+  @Test
+  void createLevelWithNull() {
+    // given
+    String null_로그_레벨 = null;
+
+    // when & then
+    assertThatThrownBy(() -> Level.from(null_로그_레벨))
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessage("level은 null이거나 빈 문자열일 수 없습니다.");
+  }
+
```

Comment:
아래와 같은 존재하지 않는 로그레벨에 대한 실패 테스트도 있으면 좋겠습니다!

```suggestion

    @DisplayName("존재하지 않는 로그 레벨 문자열을 enum으로 변환하면 예외가 발생한다.")
    @Test
    void createLevelWithNonexistentLevel() {
        // given
        String 존재하지_않는_로그_레벨 = "NONEXISTLEVEL";

        // when & then
        assertThatThrownBy(() -> Level.from(존재하지_않는_로그_레벨))
            .isInstanceOf(IllegalArgumentException.class)
            .hasMessage("No enum constant info.logbat.log.domain.enums.Level." + 존재하지_않는_로그_레벨);
    }
```

Reply:
앗! 확인했습니다 감사합니다!